tests/nl/homeassistant_HassGetState.yaml

```diff
@@ -1,2 +1,55 @@
 language: nl
-tests: []
+tests:
+  - sentences:
+      - "Wat is de buitentemperatuur"
+    intent:
+      name: HassGetState
+      slots:
+        name: "Buitentemperatuur"
+    response: "Buitentemperatuur is 15 °C"
+
+  - sentences:
+      - "Staat de slaapkamerlamp aan?"
+    intent:
+      name: HassGetState
+      slots:
+        name: "Slaapkamerlamp"
+        state: "on"
+    response: "Nee, slaapkamerlamp is off"
+
+  - sentences:
+      - "Staan er schakelaars aan in de keuken?"
+    intent:
+      name: HassGetState
+      slots:
+        area: "Keuken"
+        domain: "switch"
+        state: "on"
+    response: "Ja, Keukenschakelaar"
+
+  - sentences:
+      - "Zijn alle schakelaars aan?"
+    intent:
+      name: HassGetState
+      slots:
+        domain: "switch"
+        state: "on"
+    response: "Nee, Slaapkamerschakelaar"
+
+  - sentences:
+      - "Welke lampen staan aan?"
+    intent:
+      name: HassGetState
+      slots:
+        domain: "light"
+        state: "on"
+    response: "Werkbank, Woonkamerlamp"
```

Comment:
stel je voor dat je dit vraagt en je krijgt 20 namen te horen…

Reply:
maar goed gaan we in toekomst fixen met een style guide.

Reply:
Ik voer voor nu jouw suggestie wel door om de eerste 3 te geven en dan van de rest en count